### PR TITLE
🌱 Improve installed package error when bundle can not be found

### PR DESCRIPTION
**Pull request by m1kola**
# Description

Follow up for https://github.com/operator-framework/operator-controller/pull/499#discussion_r1381994273

## Reviewer Checklist

- [ ] API Go Documentation
- [ ] Tests: Unit Tests (and E2E Tests, if appropriate)
- [ ] Comprehensive Commit Messages
- [ ] Links to related GitHub Issue(s)

### CODE REVIEW

File: internal/resolution/variablesources/installed_package.go
Comment (m1kola) [inline internal/resolution/variablesources/installed_package.go:50]: @everettraven this is basically what you suggested [here](https://github.com/operator-framework/operator-controller/pull/499#discussion_r1382190439) but with a slighlty modified message.

I'm trying to make an emphasis on the fact that the bundle is not fund in the _currently available_ catalogs.

Let me know what you think.

P.S.: this PR will need a rebase after we merge #499.

**Review by grokspawn — APPROVED** [review]

**Comment by m1kola** [comment]
@grokspawn could you please take another look? PR got removed from a merge queue becuase #520 got merged and `variablesources/installed_package_test.go` needed an update.

**Comment by grokspawn** [comment]
> @grokspawn could you please take another look? PR got removed from a merge queue becuase #520 got merged and `variablesources/installed_package_test.go` needed an update.

Checking CI signal, and then I'll approve again if green.

**Review by grokspawn — APPROVED** [review]

**Pull request merged**
